adobe/spectrum-web-components
# docs: use alpha storybook for CEM and storysource support

## Description
- use latest storybook-built alpha release
- leverage Custom Elements Manifest as needed
- remove custom story source implementation

## Motivation and context
We should maintain less code and get more things!

## How has this been tested?

    1. Visit https://storybook--spectrum-web-components.netlify.app/storybook/?path=/docs/accordion--default
    2. Click "Show Code"
    3. See code.
    1. Visit https://storybook--spectrum-web-components.netlify.app/storybook/?path=/story/accordion--default&args=open:false
    2. Scroll through the "Controls" section and see `Properties`, `Slots`, and `Events` listed


## Types of changes

## Checklist
